### PR TITLE
Makes it so nymphs from a gestalt that splits due to damage split injured

### PR DESCRIPTION
**Pull request by Yonnimer**
What it says on the tin.

Credit to @Geevies for making the code for this.

### CODE REVIEW

**Comment by Yonnimer** [comment]
!review

File: html/changelogs/nymphsplit.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscadd: "Adds a feature where dionae who split while damaged will have their nymphs damaged."
```
Comment (Arrow768) [inline html/changelogs/nymphsplit.yml:41]: That changelog is incorrect.
The health reduction only happens on death of the diona. Not when a diona splits while damaged.

Comment (Yonnimer) [reply html/changelogs/nymphsplit.yml]: The new changelog work?

File: html/changelogs/nymphsplit.yml [hunk 2]
```diff
@@ -38,4 +38,4 @@ delete-after: True
 # Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
 # Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
 changes: 
-  - rscadd: "Adds a feature where dionae who split while damaged will have their nymphs damaged."
+  - rscadd: "Makes it so if a Dionae deathsplits, their nymphs will start damaged with only 20% of their max health."
```
Comment (Arrow768) [inline html/changelogs/nymphsplit.yml:41]:
```suggestion
  - rscadd: "Makes it so if a Dionae deathsplits, their nymphs will start with only 20% of their max health."
```

**Pull request merged**